### ISSUE TITLE
texture missing

### ISSUE THREAD

**Issue by hypervare76** [shadowmage45/KerbalFoundries2#43]
I have textures on the adjustable landing gear but thats about it with this mod everything else is grey with no texture

**Comment by shadowmage45** [comment]
Did you install TexturesUnlimited?  It is now a required dependency.

**Comment by hypervare76** [comment]
yes I have it and its probably my fault what I did is I placed whatever I installed into the game data folder and whenever I open KSP it asks me to change it too directx11 but when I do that the game starts missing textures so I got rid of directx11 entirely, and I did use one of the wheels and it was really messed up. The mod comes with TexturesUnlimited already anyway I saw it 000_TexturesUnlimited. Could it be another mod possibly? The physics on the wheels is also messed up like it bounces I saw the issue about it well I was looking through issues (Sorry to bother you)
![20190321100840_1](https://user-images.githubusercontent.com/48800866/54770990-9a3d7100-4bc1-11e9-88c3-deed56a665ff.jpg)

**Comment by hypervare76** [comment]
I will just uninstall the mod entirely. sorry for the inconvenience  
Regards, Hypervare76

**Comment by shadowmage45** [comment]
Judging by the picture something is definitely missing in your install or otherwise not loading properly -- notably the TexturesUnlimited plugin.   Yes, that is the 000_TexturesUnlimited folder.

If you are still interested in trying to solve the problem, you could provide a KSP.log file (you can drag+drop into the text box to upload), that will more than likely point towards whatever the error is.

Alternatively, you could try the older release of the mod, it still works fine with KSP 1.6, and doesn't require TexturesUnlimited or DX11 -- https://github.com/shadowmage45/KerbalFoundries2/releases/tag/2.2.6.16

**Comment by hypervare76** [comment]
Thanks sorry for no reply I will do it later today.
